tools/validators/instance_validator/tests/fake_instances/GOOD/update_no_operation_default_export.yaml

```diff
@@ -35,12 +35,12 @@ PHYSICAL-ENTITY-GUID:
   type: HVAC/CHWS_WDT
   code: PHYSICAL-ENTITY
   etag: a56790
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.